scripts/cli/install.sh

```diff
@@ -21,15 +21,15 @@ selectCliFile () {
     fileName="agc"
 
     if [[ -z "$archKind" ]]; then
-        eval $filename="agc"
+        eval $fileName="agc"
     else
-        eval $fileName="$filename-$archKind"
+        eval $fileName="$fileName-$archKind"
```

Comment:
I think these should be `fileName="agc"` and `fileName="$fileName-$archKind"`

Reply:
Testing removing the eval, but I do need to keep an eye on POSIX as well. Will have an answer shortly.